static_src/components/notification.jsx

```diff
@@ -0,0 +1,80 @@
+
+import React from 'react';
+import keymirror from 'keymirror';
+
+import Action from './action.jsx';
+import createStyler from '../util/create_styler';
+import { entityHealth } from '../constants.js';
+import style from 'cloudgov-style/css/cloudgov-style.css';
+
+const STATUSES = Object.assign({}, entityHealth, keymirror({
+  info: null
+}));
+
+
+const propTypes = {
+  message: React.PropTypes.string,
+  status: React.PropTypes.oneOf(Object.keys(STATUSES)),
+  actions: React.PropTypes.arrayOf(React.PropTypes.object),
```

Comment:
Let me clarify, if you made it type `node`, you could do this

```javascript
const actions = (
  <span>
    <Action clickHandler={ handleRefresh }>Refresh</Action>
    <Action clickHandler={ handleCancel }>Cancel</Action>
  </span>
);
<Notification actions={ actions } />
```

Arrays might also count as `node`, so you wouldn't even need the `span` in that case. I'm not sure.